config.py

```diff
@@ -15,7 +15,6 @@
 
 yelpSearchCategories = ["active",
```

Comment:
I don't think this is all the top-level categories - but maybe they're all the ones we care about? Antlam added a list of grouped categories in the Engineering doc, under Jan 27 > Second Pass. If it's too many, then we should let him know that some categories are omitted. Or we this could be a good test of the "expand" script :P